docs/src/main/sphinx/release/release-420.md

```diff
@@ -0,0 +1,66 @@
+# Release 420 (21 June 2023)
+
+## General
+
+* Add support for {func}`any_value` aggregation function. ({issue}`17777`)
+* Add support for underscores in numeric literals. ({issue}`17776`)
+* Add support for hexadecimal, binary, and octal numeric literals. ({issue}`17776`)
+* Deprecate the `dynamic-filtering.small-broadcast.*` and
+  `dynamic-filtering.large-broadcast.*` configuration properties in favor of
+  `dynamic-filtering.small.*` and `dynamic-filtering.large.*`. ({issue}`17831`)
+* Disallow using the root directory of an S3 bucket as a table location without
```

Comment:
This is not a global thing from what I understand .. at least not for exchange manager config for example .. see the PR